### DISCUSSION TITLE
Running WaveWatch with multiple nests embedded

### DISCUSSION THREAD

**Discussion by bricheno** [NOAA-EMC/WW3#1197] [General]
Hello wave modellers,

Is there a way to configure multiple individual nests?

I can add the nests in the parent model through ww3_grid.nml like this:

&OUTBND_COUNT_NML
  OUTBND_COUNT%N_LINE = 3652
/
&OUTBND_LINE_NML
OUTBND_LINE(1)=
.... etc

which writes everything out to nest1.ww3. This works fine for one embedded child model.

Next, I extend the list of OUTBND_LINE to cover 2 different domains.
but when I try to pick-up the lateral boundary data from the parent to the child, it errors like this:

*** WAVEWATCH III ERROR IN W3IOBC :
     POINT3599 -0.108828E+02 -0.728750E+01 NOT LOCATED IN GRID
EXTCDE MPI_ABORT, IEXIT=    12

Is this because everything is being written to nest1.ww3?
Is there a different syntax to generate a set of several separate nests (nest1 nest2 ... etc)? Which are in different regions?

Thanks for any answers!
Lucy

Dr Lucy Bricheno | Team leader - Coastal Ocean
National Oceanography Centre, Joseph Proudman Building, 6 Brownlow Street, Liverpool, L3 5DA
+44 (0)151 795 4800 | [luic@noc.ac.uk](mailto:luic@noc.ac.uk)  | @LBricheno
https://noc.ac.uk/

**Comment by ukmo-ccbunney** [comment]
Hi @bricheno ,
To output multiple nest files you need to delimit your list of output boundary points with a `-1` in the "number of points" column (the last column). This effectively terminates the current nest file and starts a new file (nestXXX+1) with the proceeding points.

It's detailed in the WW3_GRID example file in `inp/ww3_grid.inp`. Here's an excerpt:
https://github.com/NOAA-EMC/WW3/blob/e085bcfa4e735100dcb5a7185cd3cb655f0673e3/model/inp/ww3_grid.inp#L855-L866

Chris.
